react/rules/BRA.js

```diff
@@ -34,40 +34,51 @@ export default {
       label: 'BRA_cpf',
       required: true,
       mask: value => msk.fit(value, '999.999.999-99'),
-      validate: value => {
-        const cleanValue = value.replace(/[^\d]/g, '')
-
-        if (cleanValue.length !== 11) return false
-
-        const isRepeatedNum = '0123456789'
-          .split('')
-          .some(digit => digit.repeat(11) === cleanValue)
-
-        if (isRepeatedNum) return false
-
-        const firstReduce = cleanValue
-          .split('')
-          .slice(0, 9)
-          .reduce(
-            (acc, cur, index) => acc + parseInt(cur, 10) * (10 - index),
-            0
-          )
-
-        const firstDigit = ((firstReduce * 10) % 11) % 10
-
-        if (firstDigit !== cleanValue.charAt(9)) return false
-
-        const secondReduce = cleanValue
-          .split('')
-          .slice(0, 10)
-          .reduce(
-            (acc, cur, index) => acc + parseInt(cur, 10) * (11 - index),
-            0
-          )
-
-        const secondDigit = ((secondReduce * 10) % 11) % 10
-
-        return secondDigit === cleanValue.charAt(10)
+      validate: cpf => {
+        if (!cpf) {
+          return false
+        }
+
+        let sum = 0
+        let remainder
+
+        cpf = !cpf || typeof cpf !== 'string' ? '' : cpf.replace(/[^\d]+/g, '')
```

Comment:
Do not mutate you variable. 
And, if cpf isnt of string type, what you do?  Try do this verification on L38